test_that("Invalid function parameter", {
  expect_error(.is_connected_to_genie("GENIE"))
})

